azurerm_dns_zone/outputs.tf

```
@@ -0,0 +1,14 @@
+output "dns_zone_name" {
+  description = "The full DNS name of the new zone."
+  value       = var.dns_zone_name
+}
+
+output "dns_zone_id" {
```

```suggestion
output "id" {
```
may be just id ? See the comment above